packages/@webex/internal-plugin-metrics/src/metrics.types.ts

```diff
@@ -69,3 +76,8 @@ export type MetricEventNames =
 export type ClientType = NonNullable<RawEvent['origin']['clientInfo']>['clientType'];
 export type SubClientType = NonNullable<RawEvent['origin']['clientInfo']>['subClientType'];
 export type NetworkType = RawEvent['origin']['networkType'];
+
+export type MediaQualityEventAudioSetupDelayPayload =
```

Comment:
if you think `MediaQualityEventAudioSetupDelayPayload` is too verbose, then you can use `MQEAudioSetupDelayPayload` as well.

Reply:
I'm find with the long name.